lib/extensions/string_iec.rb

```diff
@@ -0,0 +1,12 @@
+class String
+  IEC_SIZE_SUFFIXES = %w(Ki Mi Gi Ti).freeze
+  # https://github.com/jason-riddle/deploying-with-kubernetes/wiki/Conversion-Units
+  def parse_iec_number
```

Comment:
Also, all of our core class extensions are in gems/pending/util/extensions.  lib/extensions is for extensions to Rails classes.